3p/integration.js

```diff
@@ -311,6 +312,11 @@ import {zen} from '#ads/vendors/zen';
 import {zergnet} from '#ads/vendors/zergnet';
 import {zucks} from '#ads/vendors/zucks';
 
+user().error(
```

Comment:
I think this will halt execution, is that what we want?

Reply:
https://github.com/ampproject/amphtml/blob/main/src/log.js#L315 `user().error` shouldn't throw. `user().assert` will. But you reminded me that this will trigger log entry, and might overstress AMP logging system. Changing to warn only.

Reply:
Ah TIL thanks